Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -604,7 +632,7 @@ function vtkRenderWindowInteractor(publicAPI, model) {
   // do not want extra renders as the make the apparent interaction
   // rate slower.
   publicAPI.render = () => {
-    if (model.requestAnimationCount === 0) {
+    if (model.animationRequest === null) {
```

Comment:
are you sure you want to compare against null?

Reply:
:man_shrugging:

Reply:
Could also do `animationRequesters.size === 0`

Reply:
I don't know but just wanted to mention that an object can be `null`, `undefined` and `undefined === null => false`

Reply:
I see. Then should be all good:
- init: https://github.com/Kitware/vtk-js/pull/575/files/3b786ece3a26aec42bba6ede6ee8768b6754a21f#diff-f698a8e6250bde65d1ae6422124e9a84R870
- deleted: https://github.com/Kitware/vtk-js/pull/575/files/3b786ece3a26aec42bba6ede6ee8768b6754a21f#diff-f698a8e6250bde65d1ae6422124e9a84R291